### PR TITLE
feat(pci.project.quota): add autoscaling

### PR DESCRIPTION
**Pull request by Jisay**
ref: MANAGER-6563

Signed-off-by: Jisay <jean-christophe.alleman@corp.ovh.com>

| Question         | Answer
| ---------------- | ---
| Branch?          | `develop` for bug fixes 
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | MANAGER-6563
| License          | BSD 3-Clause

## Description

Add manual control of quota autoscalling in project quota management page.

### CODE REVIEW

File: packages/manager/modules/pci/src/projects/project/quota/translations/Messages_fr_FR.json [hunk 1]
```diff
@@ -17,5 +17,10 @@
   "pci_projects_project_quota_instance": "Serveurs",
   "pci_projects_project_quota_core": "vCPU",
   "pci_projects_project_quota_ram": "RAM",
-  "pci_projects_project_quota_open_new_tab": "Ouvre un nouvel onglet"
+  "pci_projects_project_quota_open_new_tab": "Ouvre un nouvel onglet",
+  "pci_projects_project_quota_autoscaling": "Quota autoscalling",
```
Comment (marie-j) [inline packages/manager/modules/pci/src/projects/project/quota/translations/Messages_fr_FR.json:21]:
```suggestion
  "pci_projects_project_quota_autoscaling": "Quota autoscaling",
```

File: packages/manager/modules/pci/src/projects/project/quota/translations/Messages_fr_FR.json [hunk 2]
```diff
@@ -17,5 +17,10 @@
...
+  "pci_projects_project_quota_autoscaling_on": "Activé",
+  "pci_projects_project_quota_autoscaling_off": "Désactivé",
+  "pci_projects_project_quota_autoscaling_error": "Une erreur est survenue lors de la mise à jour du quota autoscalling.",
```
Comment (marie-j) [inline packages/manager/modules/pci/src/projects/project/quota/translations/Messages_fr_FR.json:24]:
```suggestion
  "pci_projects_project_quota_autoscaling_error": "Une erreur est survenue lors de la mise à jour du quota autoscalé.",
```

**Review by cbourgois — DISMISSED** [review]

**Review by mohammed-zahaf — APPROVED** [review]

**Review by JeremyDec — APPROVED** [review]

**Review by antleblanc — DISMISSED** [review]

**Comment by antleblanc** [comment]
@Jisay Could you please resolve existing conflict?
Thank you!

**Comment by antleblanc** [comment]
ping @Jisay

**Comment by marie-j** [comment]
Rebased on `develop` to clean commit history and fix conflicts

**Review by JeremyDec — APPROVED** [review]

**Pull request merged**